catalog/plugin/catalog-plugin-clientinfo/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+    <bean id="clientInfoInjectorPlugin"
+          class="org.codice.ddf.catalog.plugin.clientinfo.ClientInfoInjectorPlugin"/>
+    <service ref="clientInfoInjectorPlugin"
+             interface="org.codice.ddf.catalog.plugin.PrePolicyPlugin" ranking="90"/>
```

Comment:
Why did you set a ranking of 90 on this plugin?  The default ranking should work.